pkg/proxy/ipvs/proxier.go

```diff
@@ -322,21 +322,6 @@ func (r *realIPGetter) BindedIPs() (sets.String, error) {
 // Proxier implements proxy.Provider
 var _ proxy.Provider = &Proxier{}
 
-// parseExcludedCIDRs parses the input strings and returns net.IPNet
-// The validation has been done earlier so the error condition will never happen under normal conditions
-func parseExcludedCIDRs(excludeCIDRs []string) []*net.IPNet {
-	var cidrExclusions []*net.IPNet
-	for _, excludedCIDR := range excludeCIDRs {
-		_, n, err := net.ParseCIDR(excludedCIDR)
-		if err != nil {
-			klog.Errorf("Error parsing exclude CIDR %q,  err: %v", excludedCIDR, err)
```

Comment:
This changes the behaviour since once it finds a wrong cidr it stops processing the others, however, with this function it just continues.

https://github.com/kubernetes/utils/blob/fddb29f9d0095a7a37582a1d86732ae83620f2e8/net/net.go#L35 

 I think that @andrewsykim and @lbernail are the ones more familiar with the ipvs details, I really don't have context here

Reply:
Interesting, in the proxy config [validation](https://github.com/kubernetes/kubernetes/blob/d6a9061cb43ab98054f0f455115d87e1b20f7971/pkg/proxy/apis/config/validation/validation.go#L51), actually it already checked the error for excluded CIDRs [(here](https://github.com/kubernetes/kubernetes/blob/d6a9061cb43ab98054f0f455115d87e1b20f7971/pkg/proxy/apis/config/validation/validation.go#L152)). It seems like a duplication logic here and we can assume the CIDRs here are just valid.

Reply:
ahh good catch